test/go/iterative_verifier_collation_test.go

```diff
@@ -1,118 +0,0 @@
-package test
```

Comment:
Replaced in test/integration/inline_verifier_test.rb: https://github.com/Shopify/ghostferry/blob/b8147c5ff8653099f6c5e24128aa6fd41ff401e1/test/integration/inline_verifier_test.rb#L242-L264